packages/replay-next/components/inspector/HTMLExpandable.tsx

```diff
@@ -22,20 +24,29 @@ export default function HTMLExpandable({
   before = null,
   defaultOpen = false,
   object,
+  path,
   pauseId,
   protocolValue,
 }: {
   before?: ReactNode;
   defaultOpen?: boolean;
   object: ProtocolObject;
+  path?: string;
```

Comment:
Kind of a nit: but why is this called "persistence key" some places and "path" others? Might be nice to name it consistently.